src/index.ts

```diff
@@ -3,7 +3,7 @@
 * SPDX-License-Identifier: Apache-2.0
 **********************************************************************/
 
-import { logger as log } from './utils/logger'
+import { logger as log } from './logging/logger'
```

Comment:
update references in the fille to use logger instead of log

Reply:
Updated